server/recipeAPI/spoonacularAPI/SpoonacularRecipeAPI.ts

```diff
@@ -270,7 +275,9 @@ export default class SpoonacularRecipeAPI extends SpoonacularAPI implements IRec
             id: id,
             name: name,
             category: category,
-            nutrients: nutrients
+            nutrients: nutrients,
+            quantityUnits: [ quantity.unit ],
```

Comment:
Ingredients in recipe have only one quantity "metric" available to them. That's metric provided by recipe provider as "unit". Ingredient object on the other hand stores array of quantityUnits. Since there is only one quantityUnit that we can parse from recipe ingredient, I simply wrapped it around into array.

Reply:
I see